src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_AMOUNT = FigureName.values().length;
+    private static final int MAX_SIZE = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    Random random = new Random();
```

Comment:
The 'random' field should be private to encapsulate the internal structure of the class.